buildScript/dep-init.gincl

```diff
@@ -71,6 +75,7 @@ Path jest = Paths.get("${rootDir.path}/__jest__");
 if (!Files.exists(jest)) {
     try {
 // creates __jest__ symlink
+        Files.deleteIfExists(jest)
```

Comment:
`Files.deleteIfExists(file)` seems unnecessary in all these cases because it's inside the `if (!Files.exists(file))` clause. Am I missing something?

Reply:
> `Files.deleteIfExists(file)` seems unnecessary in all these cases because it's inside the `if (!Files.exists(file))` clause. Am I missing something?

After running builds using `firefly-builder` from scratch, it creates dead links on the host.  i.e.  `/cm/irsa-ife/../firefly/node_modules`.  The `.exists` check fail when the link is dead, but the `.deleteIfExists` will remove it.  This is to fix the case when you run the build from `firefly-builder` and still want to be able to build on your host as well.